Wrappers/Python/test/test_Operator.py

```diff
@@ -97,7 +98,35 @@ def test_FiniteDifference(self):
         G.adjoint(u, out=res)
         w = G.adjoint(u)
         self.assertNumpyArrayEqual(res.as_array(), w.as_array())
+    def test_PowerMethod(self):
         
+        N, M = 2, 3
```

Comment:
both versions are ok,  add seed for the random_int and increase iterations and data size:
replace with
```
N, M = 20, 30 

ig = ImageGeometry(N, M)
Id = Identity(ig)

G = Gradient(ig)



uid = Id.domain_geometry().allocate(ImageGeometry.RANDOM_INT, seed =10)

a = LinearOperator.PowerMethod(Id, 10, uid)
b = LinearOperator.PowerMethodNonsquare(Id, 10, uid)

print ("Edo impl", a[0])
print ("old impl", b[0])

#self.assertAlmostEqual(a[0], b[0])
#self.assertNumpyArrayAlmostEqual(a[0],b[0],decimal=6)

a = LinearOperator.PowerMethod(G, 1000, uid)
b = LinearOperator.PowerMethodNonsquare(G, 1000, uid)

print ("Edo impl", a[0])
print ("old impl", b[0])

```